plugins/woocommerce-admin/client/shipping/shipping-tour.tsx

```diff
@@ -0,0 +1,163 @@
+
+const REVIEWED_DEFAULTS_OPTION =
+	'woocommerce_admin_reviewed_default_shipping_zones';
+
+const CREATED_DEFAULTS_OPTION =
+	'woocommerce_admin_created_default_shipping_zones';
+
+const useShowShippingTour = () => {
+	const {
+		hasCreatedDefaultShippingZones,
+		hasReviewedDefaultShippingOptions,
+		isLoading,
+	} = useSelect( ( select ) => {
+		const { hasFinishedResolution, getOption } =
+			select( OPTIONS_STORE_NAME );
+
+		return {
+			isLoading:
+				! hasFinishedResolution( 'getOption', [
+					CREATED_DEFAULTS_OPTION,
+				] ) &&
+				! hasFinishedResolution( 'getOption', [
+					REVIEWED_DEFAULTS_OPTION,
+				] ),
+			hasCreatedDefaultShippingZones:
+				getOption( CREATED_DEFAULTS_OPTION ) === 'yes',
+			hasReviewedDefaultShippingOptions:
+				getOption( REVIEWED_DEFAULTS_OPTION ) === 'yes',
+		};
+	} );
+
+	return {
+		isLoading,
+		show:
+			! isLoading &&
+			hasCreatedDefaultShippingZones &&
+			! hasReviewedDefaultShippingOptions,
+	};
+};
+
+export const ShippingTour = () => {
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+	const { show: showTour } = useShowShippingTour();
+	const activePlugins = useSelect( ( select ) =>
+		select( PLUGINS_STORE_NAME ).getActivePlugins()
+	);
+
+	const tourConfig: TourKitTypes.WooConfig = {
+		placement: 'auto',
+		steps: [
+			{
+				referenceElements: {
+					desktop: 'table.wc-shipping-zones',
+				},
+				meta: {
+					name: 'shipping-zones',
+					heading: __( 'Shipping zones', 'woocommerce' ),
+					descriptions: {
+						desktop: (
+							<>
+								<span>
+									{ __(
+										'We added a few shipping zones for you based on your location, but you can manage them at any time.',
+										'woocommerce'
+									) }
+								</span>
+								<br />
+								<span>
+									{ __(
+										'A shipping zone is a geography area where a certain set of shippping methods are offered.',
+										'woocommerce'
+									) }
+								</span>
+							</>
+						),
+					},
+				},
+			},
+			{
+				referenceElements: {
+					desktop: 'table.wc-shipping-zones',
+				},
+				meta: {
+					name: 'shipping-methods',
+					heading: __( 'Shipping methods', 'woocommerce' ),
+					descriptions: {
+						desktop: __(
+							'We defaulted to some recommended shippping methods based on your store location, but you can manage them at any time within each shipping zone settings.',
+							'woocommerce'
+						),
+					},
+				},
+			},
+		],
+		closeHandler: () => {
+			updateOptions( {
+				[ REVIEWED_DEFAULTS_OPTION ]: 'yes',
+			} );
+		},
+	};
+
+	const WCS_LINK_SELECTOR = 'a[href*="woocommerce-services-settings"]';
+	const isWcsSectionPresent = document.querySelector( WCS_LINK_SELECTOR );
+
+	const SHIPPING_RECOMMENDATIONS_SELECTOR =
+		'div.woocommerce-recommended-shipping-extensions';
+	const isShippingRecommendationsPresent = ! activePlugins.includes(
+		'woocommerce-services'
+	);
+
+	if ( isWcsSectionPresent ) {
```

Comment:
I think in the future, we might want to have this code provided by the WooCommerce Shipping plugin itself. That way the WooCommerce Shipping plugin can iterate on it directly without waiting on the Core release train, for example if the plugin name changes again in the future.

So maybe a future next step is to bring in an integration point for the Shipping plugin (or any other shipping plugin?) to add their banner and tour step in a standardized way.

Reply:
Good idea yea I think we should make it standard to have a filter for adding tour kit steps